script.js

```diff
@@ -1,11 +1,57 @@
 
-getEverything({q: 'pole vault', source: 'CNN'}).then((response) => {
+getEverything({ q: 'pole vault', source: 'CNN' }).then((response) => {
   const { articles } = response;
+
+  // var articles = response.articles;
   const { url } = articles[0];
-  console.log({url, googleResults: response})
+
+
+  // var url = articles[0].url;
+  // console.log({ url, googleResponse: response })
+
+  renderTitles(articles)
+
   return getAnalysis({ url });
 }).then((response) => {
-  console.log({response})
+
+
 })
 
-// https://api.aylien.com/api/v1/combined?url=https://mashable.com/article/doomsday-vault-cherokee-nation/&endpoint=sentiment&endpoint=summarize
\ No newline at end of file
+// https://api.aylien.com/api/v1/combined?url=https://mashable.com/article/doomsday-vault-cherokee-nation/&endpoint=sentiment&endpoint=summarize
```

Comment:
We can delete this. It was put there as a reference, but serves no function anymore.